tests/acceptance/features/lib/FilesPageElement/SharingDialog.php

```diff
@@ -402,6 +403,33 @@ public function shareWithUser(
 		);
 	}
 
+	/**
+	 *
+	 * @param string|null $name
+	 * @param string|null $nameToMatch
+	 * @param Session $session
+	 * @param boolean $quiet
+	 * @param int $maxRetries
+	 *
+	 * @return void
+	 * @throws ElementNotFoundException|Exception
+	 */
+	public function shareWithUsers(
+		?string  $name,
+		?string $nameToMatch,
+		Session $session,
```

Comment:
Should `$name` be optional? And optional params should come after compulsory ones 
```suggestion
	public function shareWithUsers(
		string  $name,
		string $nameToMatch,
		Session $session,
```